docs/source/upgrading_your_components.md

```diff
@@ -0,0 +1,208 @@
+
+At a high level, upgrading the binary level of your nodes is a two step process:
+
+1. Backup the ledger and MSPs.
+2. Upgrade binaries to the latest version.
+
+If you own both ordering nodes and peers, it is a best practice to upgrade the ordering nodes first. If a peer falls behind or is temporarily unable to process certain transactions, it can always catch up. If enough ordering nodes go down, by comparison, a network can effectively cease to function.
+
+This topic presumes that these steps will be performed using CLI commands. If you are utilizing a different deployment method (Rancher, Kubernetes, OpenShift, etc) consult their documentation on how to use their CLI.
+
+Note that for native and production deployments, you will also need to update the YAML configuration file for the nodes (for example, the `orderer.yaml` file) with the one from the release artifacts.
+
+To do this, backup the `orderer.yaml` or `core.yaml` file (for the peer) and replace it with the `orderer.yaml` or `core-yaml` file from the release artifacts. Then port any modified variables from the backed up `orderer.yaml` or `core.yaml` to the new one. Using a utility like `diff` may be helpful. Note that updating the YAML file from the release rather than updating your old YAML file **is the recommended way to update your node YAML files**, as it reduces the likelihood of making errors.
+
+We will talk more about these configuration files in the relevant steps.
+
+### Ledger backup and restore
+
+While we will demonstrate the process for backing up ledger data in this tutorial, it is not strictly required to backup the ledger data of a peer or an ordering node (assuming the node is part of a larger group of nodes in an ordering service). This is because, even in the worst case of catastrophic failure of a peer (such as a disk failure), the peer can be brought up with no ledger at all. You can then have the peer re-join the desired channels and as a result, the peer will automatically create a ledger for each of the channels and will start receiving the blocks via regular block transfer mechanism from either the ordering service or the other peers in the channel. As the peer processes blocks, it will also build up its state database.
+
+However, backing up ledger data enables the restoration of a peer without the time and computational costs associated with bootstrapping from the genesis block and reprocessing all transactions, a process that can take hours (depending on the size of the ledger). In addition, ledger data backups may help to expedite the addition of a new peer, which can be achieved by backing up the ledger data from one peer and starting the new peer with the backed up ledger data.
+
+This tutorial presumes that the file path to the ledger data has not been changed from the default value of `/var/hyperledger/production/<node>`. If this location has been changed for your nodes, enter the path to the data on your ledgers in the commands below.
```

Comment:
@joealewine: Default file ledger location path for peer is `/var/hyperledger/production/` and orderer is `/var/hyperledger/production/orderer`. Location mentioned `/var/hyperledger/production/<node>` should be able to differentiate for peer and orderer ledger locations.